app/digests/tests/test_digest_api.py

```diff
@@ -38,13 +42,19 @@ def test_has_expected_data_in_response(client: Client,
 def test_can_get_digest_by_id(client: Client,
                               digest: Digest,
                               digest_json: Dict):
-    response = client.get(f'/digests/{digest.id}')
+    response = client.get(f'{DIGESTS_URL}/{digest.id}')
     assert response.data == digest_json
     assert response.content_type == settings.DIGESTS_CONTENT_TYPE
 
 
 @pytest.mark.django_db
 def test_has_digest_content_type_header(client: Client):
-    response = client.get('/digests')
+    response = client.get(DIGESTS_URL)
     assert response.status_code == 200
     assert response.content_type == settings.DIGESTS_CONTENT_TYPE
+
+
+@pytest.mark.django_db
+def test_can_create_digest(rest_client: APIClient, digest_json: Dict):
+    response = rest_client.post(DIGESTS_URL, data=digest_json, format='json')
+    assert response.status_code == 201
```

Comment:
@gnott surely needs to review this call, which should be performed by the bot. @seanwiseman can you point to where the `digest_json` sample input is?

Reply:
we often use the words `ingest` and `publish` to refer to the two stages of creating an item and then making it available to the world later

Reply:
`digest_json` is with the fixtures in [`conftest.py`](https://github.com/elifesciences/digests/blob/92cc4410e10dc01d373c13e39c63e2daa5a9f1f0/app/digests/tests/conftest.py#L164)